### ISSUE TITLE
User should be able to use ENTER to focus submit button

### ISSUE THREAD

**Issue by jorisleker** [kiesraad/abacus#306]
When the user has focus on the last input field on the page, and presses `ENTER`, focus should be set to the 'Volgende' button. Right now, focus is set to the first input on the page. 

Result of this change will be that it will be possible to submit a form section by pressing `ENTER` twice after completing the last input on the page.

**Comment by jorisleker** [comment]
duplicate of #207
